fastlane/jetpack_metadata/tr/subtitle.txt

```diff
@@ -1 +0,0 @@
-Daha hızlı, güvenli WordPress
```

Comment:
⚠️  Remember to check that one before final release too (just like you said for `es-ES` above)